x-pack/platform/plugins/private/upgrade_assistant/server/lib/es_deprecations_status/migrations.ts

```diff
@@ -191,32 +191,6 @@ const enrichIndexSettingsDeprecations = async (
   });
 };
 
-const excludeDeprecation = (
```

Comment:
Seems that removing this function makes sense, re-reading these now its clear they are nonsense. And specifically for the previously added `autoFix` check, that was for autocorrecting apm v7 indices in 8, so dont need that in 9.x (see [PR](https://github.com/elastic/kibana/pull/221800)) so its alright that is not here.